superset/initialization/__init__.py

```diff
@@ -331,13 +344,14 @@ def init_views(self) -> None:
             category_icon="fa-flask",
             icon="fa-flask",
             category="SQL Lab",
-            category_label=__("SQL Lab"),
+            category_label=__("SQL"),
```

Comment:
Does this change impact the `SQLLAB_PERMISSION_VIEWS` and `ADMIN_ONLY_VIEW_MENUS` in the security manager?

Reply:
it shouldn't i'm only changing the arrangement and Labels